src/Symfony/Bridge/PhpUnit/SymfonyExtension.php

```diff
@@ -131,6 +149,6 @@ public static function hasGroup(Test $test, string $groupName): bool
             }
         }
 
-        return false;
+        return [] !== $reader->forClassAndMethod($test->className(), $test->methodName(), $attribute);
```

Comment:
```suggestion
        return [] !== $this->reader->forClassAndMethod($test->className(), $test->methodName(), $attribute);
```

Would that be possible if the `disableClockMock()`, `disableDnsMock()` and `hasGroup()` methods were not `static`? Would that simplify the code?

Reply:
@xabbuh Yes, it would. However, the main problem here is accessing `$this` of `SymfonyExtension` inside the anonymous classes. You would still need to pass it somehow, e.g. through the constructor:

```php
$facade->registerSubscriber(new class($this) implements SkippedSubscriber {
	public function __construct(private SymfonyExtension $extension)
	{
	}

	public function notify(Skipped $event): void
	{
		$this->extension->disableClockMock($event->test());
		$this->extension->disableDnsMock($event->test());
	}
});
```

Reply:
ah indeed, not worth it